reference_guide/intellij_artifacts.md

```diff
@@ -1,100 +1,69 @@
 ---
-title: IntelliJ Platform Artifacts Repository
+title: IntelliJ Platform Artifacts Repositories
 ---
 
-JetBrains maintains a public repository that hosts artifacts related to the IntelliJ Platform, such as binaries and source code. This 
-repository makes artifacts more accessible for plugin authors.
+JetBrains maintains public repositories that host artifacts related to the IntelliJ Platform, such as binaries and source code. These 
+repositories make artifacts more accessible for plugin developers.
 
-The IntelliJ Platform Artifacts Repository contains:
+The IntelliJ Platform artifacts repositories are:
 * [Releases repository](https://www.jetbrains.com/intellij-repository/releases/) for release and EAP versions.
 * [Snapshots repository](https://www.jetbrains.com/intellij-repository/snapshots/) for EAP candidates and the latest EAP for each branch.
+
+Both the Releases and Snapshots repositories have two types of content:
+* Artifacts for cross-platform, ZIP distributions of IntelliJ IDEA binaries and source code.
+These artifacts are not intended to be accessed directly from a plugin project's `build.gradle` file. 
+The `gradle-intellij-plugin` will access them as-needed for a plugin project.
+* Artifacts for individual modules from the IntelliJ Platform. 
+These may be downloaded, or accessed directly from a `build.gradle` file, as explained below.
   
-[Dependencies](https://jetbrains.bintray.com/intellij-third-party-dependencies) of individual modules from the IntelliJ
-Platform are hosted at the bintray repository. These artifacts should not be used directly, but a link to this 
-repository should be added to pom.xml/build.gradle files if individual modules from the IntelliJ Platform Artifacts Repository are used.    
+Artifacts for IntelliJ Platform third-party dependencies are hosted at the [Bintray repository](https://jetbrains.bintray.com/intellij-third-party-dependencies). 
+A link to this repository should be added to `pom.xml`/`build.gradle` files when individual modules from an IntelliJ Platform artifacts repository are used.    
 
-## How to Use the IntelliJ Platform Artifacts Repository
-Artifacts in the repositories are utilized by adding information to a project's build.gradle file. See
-the [gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin) for more information about
-Gradle support. 
+## Using IntelliJ Platform Module Artifacts
+IntelliJ Platform module artifacts are utilized by adding information to a project's `build.gradle` file. 
+See the [gradle-intellij-plugin README file](https://github.com/JetBrains/gradle-intellij-plugin) for more information about IntelliJ IDEA Gradle support. 
 
 If you want to setup the dependencies manually, there are two types of information needed to use a repository:
```

Comment:
> If you want to setup the dependencies manually
We are talking about IntelliJ Platform Module artifacts here, and currently there is no way to use them automatically. So I would just skip that part.